streamly.cabal

```diff
@@ -3,30 +3,27 @@ name:               streamly
 version:            0.7.2
 synopsis:           Dataflow programming and declarative concurrency
 description:
-  <https://streamly.composewell.com Streamly> is a framework to build
-  reliable and scalable software systems from modular building blocks
-  using dataflow programming (streaming) and declarative concurrency.
-  Powerful stream fusion framework in streamly allows high-performance,
-  modular combinatorial programming. Key features include:
+  Browse the documentation at https://streamly.composewell.com.
+  .
```

Comment:
There is already a `period` on the line above.

Reply:
Hmm..its not a period. It is to tell cabal that it is a blank line.